sdcm/nemesis.py

```diff
@@ -1303,6 +1304,15 @@ def _terminate_cluster_node(self, node):
     def _nodetool_decommission(self, add_node=True):
         if self._is_it_on_kubernetes():
             self.set_target_node(allow_only_last_node_in_rack=True)
+        # If any keyspace RF equals to number-of-cluster-nodes, where tablets are in use,
+        # then a decommission is not supported and has to be skipped.
+        with self.cluster.cql_connection_patient(self.target_node) as session:
+            if is_tablets_feature_enabled(session):
+                max_rf, keyspace = get_max_replication_factor(session)
+                if max_rf == len(self.cluster.nodes):
```

Comment:
Ultimately, need to check rf vs. number of nodes in the datacenter.

Reply:
With tablet at the moment you even need to check that the test doesn't try to decommission the last node in a particular rack.

Reply:
that is theoretically right. currently SCT doesn't cover such a relevant scenario.
I think it would be better to address this in a separate issue+PR.
We may also have to address the issue for AddRemoveDC nemesis.